main.js

```diff
@@ -160,4 +160,175 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+/*---------------------------------------------------------------------
+ASSIGNMENTS
+---------------------------------------------------------------------*/
+
+/*----------------------
+    Opdracht 1 - Array Methoden
+----------------------*/
+// Opdracht 1a: Gebruik een array-methode om een array te maken met alle tv-type namen. Log de uitkomst in de console.
+const tvType = inventory.map((tv) => {
+  return tv.type;
+});
+console.log("Array met alle tv-type namen:");
+console.log(tvType);
+
+// Opdracht 1b: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die volledig uitverkocht zijn. Log de uitkomst in de console.
+const soldOut = inventory.filter((tv) => {
+  const inventoryTv = tv.originalStock - tv.sold
+  return inventoryTv === 0;
+});
+
+console.log("Tv's die volledig uitverkocht zijn:");
+console.log(soldOut);
+
+// Opdracht 1c: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die over AmbiLight beschikken. Log de uitkomst in de console.
+const ambilightTvs = inventory.filter((tv) => {
+  return tv.options.ambiLight === true;
```

Comment:
`boolean_waarde === true` is altijd gelijk aan `boolean_waarde`, dus in dit geval
```suggestion
  return tv.options.ambiLight;
```